guidelines/style-guidelines.md

```diff
@@ -0,0 +1,162 @@
+
+Passive writing mentions the object (the person or thing 'receiving' the action) first. Passive sentences often include 'by' (for example, house prices were pushed up by inflation).
+
+Active writing is more direct and should be used most of the time. We use short, clear sentences.
+
+
+## Grammar and punctuation
+
+#### Exclamation marks
+
+We don't use exclamation marks in most instances - for instance, we don't use exclamation marks in documentation or page content.
+
+#### Colon and semi-colons
+
+We avoid these. When possible we’ll write 2 sentences instead, or separate the clauses using a dash (with a space on either side).
+
+#### Apostrophes
+
+We generally don’t add an extra 's' after nouns or names ending in 's'.
+For example: "The business’ work" – not  "the business’s work."
+
+We don’t use an apostrophe for dates, numbers, or plurals of abbreviations.
+For example: 1960s.
+
+#### Bold
+
+We rarely use bold – using too much will make it difficult for users to know which parts of your content they need to pay the most attention to. To emphasise words or phrases, we:
+* put key information at the start of sentences
+* use headings
+
+#### Capitals
+
+We only use capitals for proper nouns, such as:
+* names of people, places and things, including buildings and brands
+* titles, such as: of books, albums, recordings
+
+'Types' of places are not capitalized, unless referring to a specific place.
+For example: "I like the library" - and "I like my local Library"
+
+Generally, terms are not proper nouns, so should not be capitalised. Technical terms are not proper nouns. But if a word or term is branded as a distinct thing, treat it as a proper noun.
+
+#### Headings
+
+We use sentence case so only the first letter is upper case. We never link headings.
+
+#### Quotation marks
+
+We use double quotation marks for:
+* exact quotations
+* direct speech.
+
+We use single quotation marks for:
+* technical terms (the first time it is used)
+* classification descriptors
+* titles of documents or publications
+
+
+## Links
+
+We:
+* link to relevant content on our website before we link to external websites 
+* never link headings
+* write descriptive links that tell you what you’ll find when you follow them - not 'click here'
+* don't set links to open in a new tab or window, with exceptions to be decided case by case.
+
+
+## Lists
+
+We use bulleted lists (coded as unordered lists) to list items or points, and numbered lists (coded as ordered lists) for processes where the order of steps is important.
+
+We try to:
+* keep our lists short (2–7 items)
+* only use 1 level of nesting.
+
+We use 2 types of bulleted lists – single-sentence lists and multi-sentence lists.
+
+When we’re writing a single-sentence list, we:
+* start with a stem sentence that all the points have in common
+* start each point in lower case, and only use a full stop on the last point
+* don't use 'and' or 'or' on the second-to-last point
```

Comment:
Not sure I follow this one.

Reply:
Oh, is it meant to be in the end, like don't say "blah blah blah, and" "blah"?

Reply:
Sorry, missed this comment, yes that's how I understood it